registry/session.go

```diff
@@ -224,8 +224,6 @@ func (r *Session) GetRemoteTags(registries []string, repository string, token []
 
 		if res.StatusCode == 404 {
 			return nil, fmt.Errorf("Repository not found")
-		} else if res.StatusCode != 200 {
```

Comment:
Oh! I think the original description wasn't clear on this; the actual "continue" should still be executed, but in a separate "if", (no "else if"), i.e.:

```go
if res.StatusCode != 200 {
    continue
}
```